### ISSUE TITLE
[Classic Era/SoM] Quest Tracker won't collapse/minimise at the top level

### ISSUE THREAD

**Issue by alphabitnz** [Questie/Questie#4331]
[Classic Era/SoM] Quest Tracker won't collapse/minimise at the top level

Only started with the most recent update today (v7.3.1)

Bugsack output on error condition:
```
1x ...ceQuestie\Modules\Tracker\QuestieTracker.lua:1185: attempt to call field 'Hide' (a nil value)
[string "@Questie\Modules\Tracker\QuestieTracker.lua"]:1185: in function <...ceQuestie\Modules\Tracker\QuestieTracker.lua:1179>

Locals:
self = <unnamed> {
 0 = <userdata>
 mode = 0
 SetMode = <function> defined @Questie\Modules\Tracker\ActiveQuestsHeader.lua:185
 label = <unnamed> {
 }
}
(*temporary) = nil
(*temporary) = "attempt to call field 'Hide' (a nil value)"
AchievementTracker = <table> {
 private = <table> {
 }
}
_QuestieTracker = <table> {
 CreateTrackedQuestItemButtons = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:229
 CreateTrackedQuestsFrame = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:204
 trackedQuestsFrame = Questie_TrackedQuests {
 }
 PrintProgressColor = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1222
 activeQuestsHeader = <unnamed> {
 }
 baseFrame = Questie_BaseFrame {
 }
}
QuestieTracker = <table> {
 HookBaseTracker = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1124
 MoveDurabilityFrame = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:455
 Enable = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:481
 Unhook = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1110
 Update = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:695
 UntrackQuestId = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1249
 _GetNumQuestWatches = <function> defined =[C]:-1
 private = <table> {
 }
 CheckDurabilityAlertStatus = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:469
 _last_aqw = 15
 Untrack = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1105
 UpdateQuestProximityTimer = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1435
 started = true
 _alreadyHooked = true
 _IsQuestWatched = <function> defined =[C]:-1
 _alreadyHookedSecure = true
 ResetDurabilityFrame = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:450
 AQW_Insert = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1276
 RemoveQuest = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:1204
 Collapse = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:515
 Initialize = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:79
 Expand = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:521
 Toggle = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:507
 Disable = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:495
 ResetLocation = <function> defined @Questie\Modules\Tracker\QuestieTracker.lua:435
}

```

**Comment by BreakBB** [comment]
Hey @alphabitnz can you run the following command and report the output?

`/dump Questie.IsWotlk`

**Comment by alphabitnz** [comment]
I believe there was a re-release of 7.3.1 that I can confirm did resolve this bug as per its changelog:
"Fixed Tracker errors on Classic Era"

Output of that command on the fixed 7.3.1 is:
[1]=false

It's possible I could have just been on 7.3.0 when I opened this issue, but I update via Curseforge client every day and the issue started earlier on the day that I opened this ticket after there was an update that day so make of that what you will.

Bug has been fixed now but let me know if there's anything more you need from me!
